build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
     }
+
     if ( hasHTML || hasPlain ) {
+        var range = this.getSelection();
         this.saveUndoState();
+        this.setSelection( range );
```

Comment:
At this point, the drop hasn't occurred, so the selection should not have changed. This addresses an issue I found with Internet explorer where saveUndoState was setting the selection to the end of the document. I need to investigate the issue further to determine why that was happening.